sass/_header.scss

```diff
@@ -143,76 +132,76 @@ Nav menu, which animates into close x
 --------------------------------------------------------------*/
 
 .hamburger {
-	border: 0;
-    display: inline-block;
-	height: 30px;
-    margin: 1px auto -15px;
-    padding-top: 5px;
-    -webkit-tap-highlight-color: transparent;
-    width: 28px;
-    
-    span {
-        display: block;
-        position: relative;
-        
-        &:before,
-        &:after {
-            content: '';
-            left: 0;
-            position: absolute;
-            top: -9px;
-        }
-        
-        &:after {
-            top: 9px;
-        }
+  border: 0;
+  display: inline-block;
+  height: 30px;
+  margin: 1px auto -15px;
+  padding-top: 5px;
+  -webkit-tap-highlight-color: transparent;
+  width: 28px;
+
+  span {
+    display: block;
+    position: relative;
+
+    &:before,
+    &:after {
+      content: '';
+      left: 0;
+      position: absolute;
+      top: -9px;
+    }
+
+    &:after {
+      top: 9px;
     }
+  }
 }
 
 .hamburger span,
 .hamburger span:after,
 .hamburger span:before {
-	backface-visibility: hidden;
-	background-color: #fff;
-	border-radius: 2px;
-	height: 5px;
-    transition: all .5s;
-	width: 100%;
+  backface-visibility: hidden;
+  background-color: #fff;
+  border-radius: 2px;
+  height: 5px;
+  transition: all .5s;
+  width: 100%;
 }
 
 .noscroll .hamburger,
 .hamburger.close {
-    margin-right: 18px;
-    pointer-events: auto;
-    transform: translate3d(0, 3px, 0);
-	
-    @media screen and (max-width: $width-tablet) {
-        margin-right: 0;
-    }
-    
-    &:before {
-        content: '';
-        height: 100%;
-        left: 0;
-        position: absolute;
-        top: -25%;
-        width: 100%;
-    }
-    
-	span {
-		background-color: transparent !important;
-	}
-	
-	span:before,
-	span:after {
-		background-color: #fff !important;
-	}
-	
-    span:before {
-        transform: rotate(45deg) translate(5px, 5px);
-    }
-    
-    span:after {
-        transform: rotate(-45deg) translate(7px, -8px);
-    }
+  margin-right: 18px;
+  pointer-events: auto;
+  transform: translate3d(0, 3px, 0);
+
+  @media screen and (max-width: $width-tablet) {
+    margin-right: 0;
+  }
+
+  &:before {
+    content: '';
+    height: 100%;
+    left: 0;
+    position: absolute;
+    top: -25%;
+    width: 100%;
+  }
+
+  span {
+    background-color: transparent !important;
+  }
+
+  span:before,
+  span:after {
+    background-color: #fff !important;
```

Comment:
Color literals like `#fff` should only be used in variable declarations; they should be referred to via variable everywhere else.